source/renderer/app/containers/wallet/dialogs/WalletCreateDialogContainer.js

```diff
@@ -73,7 +77,7 @@ export default class WalletCreateDialogContainer extends Component<Props> {
       createWalletStep,
       createWalletShowAbortConfirmation,
     } = this.props.stores.wallets;
-    const stepId = CREATE_WALLET_STEPS[createWalletStep];
+    const stepId = CREATE_WALLET_STEPS[createWalletStep || 0];
```

Comment:
@yakovkaravelov I'm not sure that this fallback to `0` is necessary because when the creation process is started store will set the current step to 0. In this way, you will mutate / fake values and flow provided by the store.
https://github.com/input-output-hk/daedalus/blob/9fff3ba218b9030ee68f158c25a072b670763588/source/renderer/app/stores/WalletsStore.js#L286

Reply:
This was added for flow error fix, not for above purpose. @tomothespian